packages/mint-components/src/components/sqm-scroll/sqm-scroll.tsx

```diff
@@ -0,0 +1,126 @@
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { Component, h, Prop, Host, State } from "@stencil/core";
+import { useScroll } from "./useScroll";
+import { createStyleSheet } from "../../styling/JSS";
+
+@Component({
+  tag: "sqm-scroll",
+  shadow: true,
+})
+export class Scroll {
+  @State()
+  ignored = true;
+
+  @Prop() buttonType:
```

Comment:
Missing uiNames for all props, use descriptions where necessary

Reply:
Done!